### PR TITLE
api for friend recommendations, inspired by UserCommander#getFriendRecom...

### PR DESCRIPTION
**Pull request by eishay**
...mendations and fitted to the format of new user profile cards of profile pages.

### CODE REVIEW

**Comment by eishay** [comment]
@2is10 @leogrim

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserProfileController.scala [hunk 1]
```diff
@@ -60,6 +62,32 @@ class UserProfileController @Inject() (
     }
   }
 
+  /**
+   * @param fullInfoLimit trying to get at least this number of full card info
+   * @param maxExtraIds if available, returning external id of additional maxExtraIds ids that the frontend can iterate on as it does with getProfileConnections
+   */
+  def getFriendRecommendations(fullInfoLimit: Int, maxExtraIds: Int) = UserAction.async { request =>
+    abookServiceClient.getFriendRecommendations(request.userId, 0, fullInfoLimit + maxExtraIds, true) map {
+      case None => Ok(Json.obj("users" -> JsArray()))
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserProfileController.scala:71]: @eishay Why would this be `None`? Should I handle it like a 500 (temporary server error)?

**Comment by 2is10** [comment]
:+1: Thanks!

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserProfileController.scala (hunk 1 above)
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserProfileController.scala:71]: I think it''s handling the case where the graph is late and hasn't ingested
the user yet

**Pull request merged**